### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by devalevenkatesh**
**Issue #:**
#8 

**Description of changes:**
- Update README to use Lex V1 console.

**Testing**

1. How did you test these changes? Just an README update.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by ltrung** [comment]
What is the difference between V1 and V2? Should we update the demo to use V2 in the future :)

**Comment by devalevenkatesh** [comment]
> What is the difference between V1 and V2? Should we update the demo to use V2 in the future :)

Its the UI versions from the AWS Management console for Lex (the bot service). Yes, I agree we should update it to use Lex V2 but I have to plan and update the demo as a whole as it has been not tested and updated in a while.

File: README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ The demo has a customer application that uses the pre-built customer chat widget
 #### Step 1: Create StartVideoCall Amazon Lex Bot
 This bot is needed for the contact flow (`Chime Connect Integration flow.json`). This bot helps in branching the Amazon Connect contact flow decision making when the customer wants to do a video enabled chat or just simple chat.
 
-1. Goto [Amazon Lex](https://console.aws.amazon.com/lex)
+1. Goto [Amazon Lex](https://console.aws.amazon.com/lex). Here, please use Lex V1 console if it takes you diretly to V2 console. You can go to V1 console by clicking 'Return to the V1 console'.
```
Comment (ltrung) [inline README.md:20]: Is V2 the default right now? Can we just say:
>Please use Lex V1 console. You can change to V1 console by clicking 'Return to the V1 console'.

Comment (devalevenkatesh) [reply README.md:20]: Atleast not for me, it was showing V1 itself, but I correct the sentence as suggested, thanks!

Comment (devalevenkatesh) [reply README.md]: Updated it thanks!

**Review by ltrung — APPROVED** [review]

**Pull request merged**